package.json

```diff
@@ -38,6 +38,6 @@
     ]
   },
   "devDependencies": {
-    "husky": "^7.0.0"
```

Comment:
In the hotfix I made, I addressed this, so you can remove this from the PR. I think there's an option to split this out of the pull request if you edit it. Let me know if you need help with that.